rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -146,14 +146,18 @@ extern "C" RocalTensor ROCAL_API_CALL rocalBrightness(RocalContext context, Roca
                                                       RocalTensorOutputType rocal_tensor_output_type = ROCAL_UINT8);
 
 /// Accepts U8 and RGB24 inputs
-/// \param context
-/// \param input
-/// \param shift
-/// \param is_output
+/// \param context Rocal context
+/// \param input Input Rocal tensor
+/// \param is_output is the output tensor part of the graph output
+/// \param alpha controls contrast of the image
```

Comment:
please change the comment to tensor

Reply:
Brightness augmentation is applied only for images hence we have added the comment stating it controls contrast of the image